layouts/index.html

```diff
@@ -1,18 +1,48 @@
 {{ partial "header.html" . }}
 
-<header class="frontpage">
-  <h1>&#x1f680; Camel 3.4 is here! <wbr><a href="blog/2020/06/camel34-whatsnew/">What's new?</a></h1>
-  <svg viewBox="0 0 500 70" preserveAspectRatio="none">
-    <path d="M -10,25 C 130,125 330,0 510,5" />
-    <path d="M -10,20 C 130,125 330,0 510,2" />
-    <path d="M -10,15 C 150,115 350,0 510,15" />
-    <path d="M -10,11 C 130,125 330,0 510,20" />
-    <path d="M -10,5 C 130,125 330,0 510,25" />
-  </svg>
-  <p>Camel is an <mark>open source integration framework</mark> that empowers you to quickly and easily integrate various systems consuming or producing data.
-  </p>
-  <a class="significant" href="/manual/latest/getting-started.html">Get started</a>
-</header>
+<div class="frontpage-header">
+  <div class="main-heading">
+    <h1>Apache Camel</h1>
+    <p>
+      Camel is an Open Source integration framework that empowers you
+      to quickly and easily integrate various systems consuming or producing data.
+    </p>
+    <p>
+    <a class="button-dark" href="/manual/latest/getting-started.html">Get Started</a> <a class="button-light" href="/manual/latest/faq/what-is-camel.html">What is Camel?</a>
+    </p>
+  </div>
+  <img src="./img/camel-gears.svg" />
+</div>  
+
+<div class="whats-new">
+  <h2>What's New?</h2>
+</div>
+
+<section class="frontpage columns blog-posts">
+
+{{with .Site.GetPage "taxonomy" "releases"}}
+{{range .Pages | first 3}}
```

Comment:
Do you think we could fit more than 3, perhaps by reducing the font size of the date presented? It's quite large and probably not the more important than other text.

Reply:
Fitting more than 3 in a row would be difficult near on small screens. We can make use of a carousel to display more blogs. We can display 3 by default and have arrows on both sides, allowing the user to slide and see more blogs. What do you think about this?